### PR TITLE
Supported Objects: Gong

### PR DESCRIPTION
**Pull request by Cobalt0s**

### CODE REVIEW

**Comment by laurenzlong** [comment]
Leaving this PR unmerged for now since we haven't actually shipped the Gong connector yet.

**Comment by Cobalt0s** [comment]
@laurenzlong last deep connector PR for Gong was merged.
I have added Write support for `calls` and incremental read for calls, which is reflected in updated docs. Let me know when this PR can be merged.

File: src/provider-guides/gong.mdx [hunk 1]
```diff
@@ -13,6 +13,14 @@ updatedAt: "Fri Jul 19 2024 21:54:48 GMT+0000 (Coordinated Universal Time)"
 This connector supports:
 - [Proxy Actions](/define-integrations/proxy-actions), using the base URL `https://api.gong.io`.
```
Comment (laurenzlong) [inline src/provider-guides/gong.mdx:14]: Can you revise this list to add Read Actions and Write Actions?

**Review by laurenzlong — CHANGES_REQUESTED** [review]

File: src/provider-guides/gong.mdx (hunk 1 above)
Comment (Cobalt0s) [reply src/provider-guides/gong.mdx:14]: Addressed.

I created the series of Gong PRs across repositories, where this one will be merged last. I feel positive about Gong.

File: src/provider-guides/gong.mdx [hunk 2]
```diff
@@ -12,6 +12,16 @@ updatedAt: "Fri Jul 19 2024 21:54:48 GMT+0000 (Coordinated Universal Time)"
...
+- [Read Actions](/define-integrations/read-actions), including a full historical backfill, while incremental reads are limited to calls only.
+- [Write Actions](/define-integrations/write-actions).
+
+### Supported Objects
+
+The Gong connector supports **reading** from the following objects:
+- Calls (incremental read)
+- Users
+- Workspaces
+Writing is supported for `Calls`, allowing submitting calls for AI processing.
```
Comment (laurenzlong) [inline src/provider-guides/gong.mdx:24]:
```suggestion
Writing is supported for Calls.
```

**Review by laurenzlong — APPROVED** [review]

**Comment by laurenzlong** [comment]
merging this since Gong deep connector has been shipped in https://github.com/amp-labs/connectors/pull/1064

**Pull request merged**